src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -370,7 +370,11 @@ private void ConfigureApi(IApplicationBuilder app)
                     ctx => IsGet(ctx) &&
                         ctx.Request.Path.StartsWithSegments("/api") &&
                         !ctx.Request.Path.Value.EndsWith("swagger.json"),
-                    a => { a.Run(ApiRedirectHandler); });
+                    a =>
+                    {
+                        app.UseAuthentication();
```

Comment:
We are adding authentication here?

Reply:
This was something we broke a while ago. The prod api redirect requires authentication to function, but we don't even process auth in this "local dev outside of SF" scenario, so this was 100% failing. Without this the e2e tests can't function.